### PR TITLE
Initialize Slack app

### PR DESCRIPTION
**Pull request by MathiasMoralesDixtra**
- Save all workspace users on Db when the app is installed.
- Create default team
- Create default office
- Create default room

### CODE REVIEW

**Comment by mauriguz** [comment]
@MathiasMoralesDixtra please pull latest master into this branch, I see changes that are in both PRs

**Comment by MathiasMoralesDixtra** [comment]
> @MathiasMoralesDixtra please pull latest master into this branch, I see changes that are in both PRs

Done

File: controllers/user.controller.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  const batchingUsers = async (_user) => {
+    const {
+      profile, id, tz, team_id, is_admin, is_owner,
+    } = _user;
+    const {
+      first_name, last_name, phone, email, image_72,
+    } = profile;
+
+    const role = is_admin || is_owner ? 1 : 0;
```
Comment (mauriguz) [inline controllers/user.controller.js:15]:
```suggestion
    const role = Number(is_admin || is_owner);
```

File: entities/room.entities.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const { Schema, model } = mongoose;
 const RoomSchema = new Schema({
   name: { type: String },
   enabled: { type: Boolean },
-  maxCapacity: { type: Number },
+  maxCapacity: { type: Number, default: 10 },
 });
```
Comment (mauriguz) [inline entities/room.entities.js:9]: I see that officeschema's default is `default: true`, should this schema have the same?

File: entities/user.entities.js [hunk 3]
```diff
@@ -12,10 +12,10 @@ const UserSchema = new Schema({
   email: { type: String, required: true },
   slackId: { type: String },
   role: { type: String, default: 0, index: true },
-  encryptedPassword: { type: String, required: true },
+  encryptedPassword: { type: String },
```
Comment (mauriguz) [inline entities/user.entities.js:15]: why no longer required?

File: app.js [hunk 4]
```diff
@@ -22,14 +23,18 @@ const receiver = new ExpressReceiver({
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   clientId: process.env.SLACK_CLIENT_ID,
   clientSecret: process.env.SLACK_CLIENT_SECRET,
-  token: process.env.SLACK_BOT_TOKEN,
```
Comment (mauriguz) [inline app.js:25]: no longer necessary because of all the auth stuff?

**Review by mauriguz — CHANGES_REQUESTED** [review]

File: app.js (hunk 4 above)
Comment (MathiasMoralesDixtra) [reply app.js:25]: Yes, it's not necessary now.

File: entities/user.entities.js (hunk 3 above)
Comment (MathiasMoralesDixtra) [reply entities/user.entities.js:15]: Because when I generate a user with the slackData he doesn't have a password and I can't save him.

**Review by mauriguz — APPROVED** [review]

**Pull request merged**